tests/test_enochecker.py

```diff
@@ -184,57 +166,13 @@ def test_dict():
     assert len(db) == 0
 
 
-def test_args():
```

Comment:
Did the test move somewhere else?

Reply:
yes, this uses the arg parser from `enochecker_cli` where test cases already exist